README.md

```diff
@@ -1,5 +1,8 @@
 # Knative eventing
 
+[![GoDoc](https://godoc.org/github.com/knative/eventing?status.svg)](https://godoc.org/github.com/knative/eventing)
```

Comment:
Looks like this doesn't exist yet. What do we need to do to get it there?

Reply:
I was able to get this seeded now.